modules/apps/notification/notification-rest-test/src/testIntegration/java/com/liferay/notification/rest/resource/v1_0/test/NotificationQueueEntryResourceTest.java

```diff
@@ -172,11 +201,69 @@ public void testDeleteNotificationQueueEntryBatch() throws Exception {
 		super.testDeleteNotificationQueueEntryBatch();
 	}
 
-	@Ignore
 	@Override
 	@Test
 	public void testGetNotificationQueueEntriesPage() throws Exception {
-		super.testGetNotificationQueueEntriesPage();
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, "Name", "name")));
+
+		NotificationContext notificationContext =
+			NotificationTemplateUtil.createNotificationContext(
+				TestPropsValues.getUser(), StringUtil.randomString(),
+				StringUtil.randomString(),
+				Arrays.asList(
+					NotificationRecipientSettingUtil.
+						createNotificationRecipientSetting(
+							"from", "[%CURRENT_USER_EMAIL_ADDRESS%]"),
+					NotificationRecipientSettingUtil.
+						createNotificationRecipientSetting(
+							"fromName",
+							Collections.singletonMap(
+								LocaleUtil.US, RandomTestUtil.randomString())),
```

Comment:
@fabioafreitas you can use `LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString())` here